flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartition.java

```diff
@@ -533,7 +533,7 @@ public void close() {
     }
 
     @Override
-    public ResultSubpartitionView createSubpartitionView(
+    protected ResultSubpartitionView createSubpartitionView(
```

Comment:
This is because the method is only used in internal implementations of ResultPartition. The public interface has been modified to expose IndexSet parameter instead of int.